src/CarbonAware.WebApi/src/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -7,13 +9,23 @@ public static void AddMonitoringAndTelemetry(this IServiceCollection services, I
         
         var envVars = configuration?.GetSection(CarbonAwareVariablesConfiguration.Key).Get<CarbonAwareVariablesConfiguration>();
         var telemetryProvider = GetTelemetryProviderFromValue(envVars?.TelemetryProvider);
-
+        var logger = CreateConsoleLogger(configuration);
         switch (telemetryProvider) {
             case TelemetryProviderType.ApplicationInsights:
             {
-                if (!String.IsNullOrEmpty(configuration?["ApplicationInsights_Connection_String"]))
+                if (!String.IsNullOrEmpty(configuration?["ApplicationInsights_Connection_String"])) 
                 {
+                    logger.LogInformation("Application Insights connection string found");
                     services.AddApplicationInsightsTelemetry();
```

Comment:
If calling the service with an argument is done below, it should be used here too, so avoid confusion why in one case and why not in the other. If there is no arg, AddAppl... method picks the first available from the env .